data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/beacon/MilestoneDependentTypesUtil.java

```diff
@@ -56,6 +56,28 @@ SerializableOneOfTypeDefinition<T> getSchemaDefinitionForAllMilestones(
     return builder.build();
   }
 
+  public static <T extends SszData>
+      SerializableOneOfTypeDefinition<T> getAvailableSchemaDefinitionForAllMilestones(
```

Comment:
Ohh, I didn't thought about Optionals.
How about define it back with this code to avoid duplication and return usage?
```

  public static <T extends SszData>
      SerializableOneOfTypeDefinition<T> getSchemaDefinitionForAllMilestones(
          final SchemaDefinitionCache schemaDefinitionCache,
          final String title,
          final Function<SchemaDefinitions, SszSchema<? extends T>> schemaGetter,
          final BiPredicate<T, SpecMilestone> predicate) {
    return getAvailableSchemaDefinitionForAllMilestones(
        schemaDefinitionCache,
        title,
        schemaGetter.andThen(schema -> Optional.of(schema)),
        predicate);
  }
```

Reply:
Added a wrapper